config-templates/redhat/odf-local/4.10/custom-parameters.json

```diff
@@ -109,13 +132,20 @@
     "displayname": "KMS encryption",
     "name": "kms-encryption",
     "default": "false",
+    "regex": "^(?:true|false)$",
+    "min-length": "4",
+    "max-length": "5",
+    "supported-values": ["true", "false"],
     "required": "false",
     "type": "boolean"
   },
   {
-    "description": " Your KMS instance name.",
+    "description": "Your KMS instance name. The instance name must contain only lowercase alphanumeric characters or '-' and start and end with an alphanumeric character.",
     "displayname": "KMS instance name",
     "name": "kms-instance-name",
+    "regex": "(^[a-z0-9])([\\-a-z0-9]*)([a-z0-9]$)",
+    "min-length": "1",
+    "max-length": "63",
```

Comment:
We need to add validation for `Ignore Noobaa` field as well.

Reply:
Addressed in https://github.com/IBM/ibm-satellite-storage/pull/194/commits/0d2c428c6b5c144c567dca80d58f0946df7ecdf3.